crates/core/tedge_mapper/src/aws/converter.rs

```diff
@@ -44,13 +58,44 @@ impl Converter for AwsConverter {
     }
 
     async fn try_convert(&mut self, input: &Message) -> Result<Vec<Message>, Self::Error> {
-        let () = self.size_threshold.validate(input)?;
+        self.size_threshold.validate(input)?;
```

Comment:
Did not see a threshold value for the `aws` defined here. I feel the validation here is against the `aws` supported size.

Reply:
Moved size validation to the end, so that the message is validated after it's transformed by the mapper.